src/components/Header/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react"
+import { Link } from "react-router-dom"
+import { IoMdCart } from "react-icons/io"
+import Sidebar from "./Sidebar"
+import { GiHamburgerMenu } from "react-icons/gi"
+import styles from "./Header.module.scss"
+
+const Header = () => {
+  const [open, setOpen] = useState(false)
+
+  return (
+    <header className={styles.header}>
+      <nav className={styles.main_nav}>
+        <GiHamburgerMenu
+          className={styles.menubar}
+          onClick={() => setOpen(true)}
+        />
+        <img
+          src="../../../static/images/logo_2x.png"
+          alt="SABKA_BAZAAR"
```

Comment:
provide a brief but a meaningful alt text

Reply:
Alt text changed to SABKA BAZAAR LOGO which displays company name incase of image not displayed.